### ISSUE TITLE
Getting Exception in response when using odiff library

### ISSUE THREAD

**Issue by arvindgh** [Visual-Regression-Tracker/Visual-Regression-Tracker#368]
I am using **dotnet-sdk**, and when i am working with pixelmatch or looksame libraries from the VRT it is working fine.
but after changing the image comparison library to **odiff**, getting the following error in response: 
The HTTP status code of the response was not expected (500).
Status: 500
Response: 
`{"path":"/test-runs","name":"Error","message":"","exception":{},"stack":"Error\n    at ChildProcess.<anonymous> (/node_modules/odiff-bin/odiff.js:147:13)\n    at ChildProcess.emit (events.js:327:22)\n    at maybeClose (internal/child_process.js:1021:16)\n    at Socket.<anonymous> (internal/child_process.js:443:11)\n    at Socket.emit (events.js:315:20)\n    at Pipe.<anonymous> (net.js:674:12)"}
`

Request Body:
`{
  "name": "Image name",
  "os": "Mac",
  "browser": "Chrome",
  "viewport": "800x600",
  "device": "PC",
  "branchName": "master",
  "diffTollerancePercent": 1,
  "imageBase64": "123"
}`

**Comment by pashidlos** [comment]
@arvindgh thanks for report
there is an issue with alpine docker image using odiff
https://github.com/dmtrKovalenko/odiff/issues/5
https://github.com/dmtrKovalenko/odiff/issues/55

**Comment by arvindgh** [comment]
Thanks for your response, do we have any other image for using other OS? if no, do you know how we can ignore small alignment issues using the pixelmatch or looksame library?

**Comment by pashidlos** [comment]
@arvindgh use another image comparison algorithm atm 
there is an option to create non-alpine docker images but not sure if it worth that 

if you have alignment issue it's gonna be the same with odiff
most common issue is that images are taken on different OS instead of running tests in docker

**Comment by alexey-polovinkin** [comment]
@pashidlos Hi! Are there any plans to fix the issue?

**Comment by pashidlos** [comment]
@alexey-polovinkin it's external open source library
refer to those issues 
https://github.com/dmtrKovalenko/odiff/issues/5
https://github.com/dmtrKovalenko/odiff/issues/55

**Comment by s-weigand** [comment]
An option to get around this is to just not use the `apline` node image but a `debian-slim` one.
It is only 20MB bigger and doesn't have all the issues you get due to incompatibilities with `musl-libc`.
Especially considering the size the `imageUploads` folder can grow to the 20MB extra don't seam like a big deal to me 🤷‍♀️

Here is the git diff of required changes to [Visual-Regression-Tracker/backend](https://github.com/Visual-Regression-Tracker/backend) to make it work.

```diff
diff --git a/Dockerfile b/Dockerfile
index 3537be0..76e53fb 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,5 @@
 # https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/137
-FROM node:18-alpine3.18 AS builder
+FROM node:18.20.4-bookworm-slim AS builder

 # Create app directory
 WORKDIR /app
@@ -18,7 +18,9 @@ COPY src ./src
 RUN npm run build

 # https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/137
-FROM node:18-alpine3.18
+FROM node:18.20.4-bookworm-slim
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y libssl3 && rm -rf /var/lib/apt/lists/*
 COPY --from=builder /app/node_modules ./node_modules
 COPY --from=builder /app/package*.json ./
 COPY --from=builder /app/dist ./dist
diff --git a/prisma/schema.prisma b/prisma/schema.prisma
index 2397b49..a1000cf 100644
--- a/prisma/schema.prisma
+++ b/prisma/schema.prisma
@@ -1,5 +1,6 @@
 generator client {
   provider = "prisma-client-js"
+  binaryTargets = ["native", "debian-openssl-3.0.x"]
 }

 datasource db {
```

**Comment by pashidlos** [comment]
@s-weigand sounds good to me
feel free to create PR with changes

**Comment by Maheshkale447** [comment]
Was this merged to backend service. want to try out odiff.

**Comment by pashidlos** [comment]
@Maheshkale447 should be fixed in 5.0.7  https://github.com/Visual-Regression-Tracker/backend/releases/tag/api-v5.0.7

@s-weigand thanks for contribution!
